.changelog/101.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:breaking-change
+tftypes: `(AttributePath).WithElementKeyInt()` now has an `int` parameter instead of `int64`.
+```
+
+```release-note:note
```

Comment:
Do we need a second note for this, in addition to the breaking change entry?

Reply:
Thought it might be nice to explain why the change was made in the changelog, but can remove.